transport/src/main/java/com/distkv/drpc/codec/DstCodec.java

```diff
@@ -142,7 +141,7 @@ public Object decode(byte[] data) throws CodecException {
     shortToBytes(header, pos, CodecConstants.MAGIC_HEAD);
     pos += 2;
 
-    header[pos++] = Version.VERSION_1.getValue();
+    header[pos++] = CodecConstants.Version.VERSION_1.getValue();
```

Comment:
It has the same problem as the file which you last have commited

Reply:
Discussed offline.